pkg/antctl/transform/networkpolicy/transform_response_test.go

```diff
@@ -62,65 +62,109 @@ func TestListTransform(t *testing.T) {
 		TierPriority: pointer.Int32(200),
 		Priority:     pointer.Float64(8),
 	}
+	var npD = cpv1beta.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              "d",
+			UID:               "aab",
+			CreationTimestamp: metav1.Time{Time: metav1.Now().Add(3)},
+		},
+		SourceRef: &cpv1beta.NetworkPolicyReference{
+			Name: "d",
+		},
+	}
+	var npE = cpv1beta.NetworkPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:              "e",
+			UID:               "bbb",
+			CreationTimestamp: metav1.Time{Time: metav1.Now().Add(4)},
+		},
+		SourceRef: &cpv1beta.NetworkPolicyReference{
+			Name: "e",
+		},
+	}
 
 	var npList = &cpv1beta.NetworkPolicyList{
 		Items: []cpv1beta.NetworkPolicy{npA, npC, npB},
 	}
 
+	var npListK8s = &cpv1beta.NetworkPolicyList{
```

Comment:
I am not sure I understand the naming here, why is one list called `npList` and the other `npListK8s`, as opposed to `npList1` and `npList2`? What does the K8s suffix stand for?

Reply:
I was thinking `npList2` has K8s NPs to test the sort, which `npList1` hasn't. I have indicated this difference in the test case names instead, and changed to `npList1` and `npList2`.